metadata.py

```diff
@@ -0,0 +1,67 @@
+"""
+The purpose of this file is to define the metadata of the app with minimal imports. 
+
+DO NOT CHANGE the name of the file
+"""
+
+from mmif import DocumentTypes, AnnotationTypes
+
+from clams.appmetadata import AppMetadata
+
+
+# DO NOT CHANGE the function name 
+def appmetadata() -> AppMetadata:
+    """
+    Function to set app-metadata values and return it as an ``AppMetadata`` obj.
+    Read these documentations before changing the code below
+    - https://sdk.clams.ai/appmetadata.html metadata specification. 
+    - https://sdk.clams.ai/autodoc/clams.appmetadata.html python API
+    
+    :return: AppMetadata object holding all necessary information.
+    """
+    
+    # Basic Information
+    metadata = AppMetadata(
+        name="Chyron Recognition",
+        description="This tool detects chyrons and generates time segments.",
+        app_license="MIT",  
+        identifier="chyron-recognition",
+        url="https://github.com/clamsproject/app-chyronrecognition",
+    )
```

Comment:
One of not-so-obvious rules we are using is that the repository name should match the app identifier (except the `app-` prefix). Then, `chyronrecognition` vs `chyron-recognition` ? My vote is for hyphenated one, as it's more readable.